frontend-cert/css-projects/business-card/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Digital Business Card</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="business-card">
+        <img src="flower.jpg" alt="Profile Image" class="profile-image">
```

Comment:
```suggestion
        <img src="flower.jpg" alt="" class="profile-image">
```

The alt text here is not doing anything useful. In this case, I would say this is a purely decorative image and the alt text should be empty.

Reply:
Thanks for pointing that out. I have left the implementation open to the camper.